### ISSUE TITLE
torchiva.separation should be torchiva.separate

### ISSUE THREAD

**Issue by StuartIanNaylor** [fakufaku/torchiva#4]
`python -m torchiva.separation INPUT OUTPUT`

`python -m torchiva.separate ./examples/samples/mix_reverb/103-1240-0003_1235-135887-0017.wav output.wav`

**Comment by fakufaku** [comment]
@StuartIanNaylor Thanks for reporting the typo in the README! Silly of me... 😆 

I did not have plans for quantized model, but that is actually a great idea! I will look into it, but I can't promise any timeline... I'll add it to the repo if I manage to get it to work. I am interested to try out onnx for example, but do not have any experience. If you have some and would like to help out, that would also be greatly appreciated.

I am also quite interested to hear about experience with data in the wild! I would be super curious to see your results if you would like to share something 🤗

**Comment by StuartIanNaylor** [comment]
Yeah prob can by giving you a look at https://github.com/usefulsensors/openai-whisper as [nyadla-sys](https://github.com/usefulsensors/openai-whisper/commits?author=nyadla-sys) has created various examples for exports to onnx and then tflite, so plenty of templates for the conversion but prob nyadla-sys would be a great source of info.

As for noise I tested it with https://drive.google.com/file/d/1N90DtDjcm-ejbUpqMHsJAkKYIxoy5iMG/view?usp=share_link 
which did a great job and produced https://drive.google.com/file/d/1V8Frsa3H7eCx3YuNRQnm09WlC5dMKZwG/view?usp=share_link

Ignore the glitches due to clipping as that is the test sample that should of had AGC.

I have been trying to find a good low load 2 channel BSS / Deverb for 2 years now and ultimately it should be targetted speaker BSS as guess with doubletalk the target channel will be random, but tackle that one later.

I will be watching and testing, so just ask, but nyadla-sys is the man and examples as said in above repo.

**Comment by StuartIanNaylor** [comment]
@fakufaku https://github.com/wenet-e2e/wesignal as might be of interest to what is submitted
